From: Ben Hutchings <ben@decadent.org.uk>
Date: Thu, 10 Apr 2014 01:36:51 +0100
Subject: net: Avoid ABI change in 3.2.57

net/core/skbuff.c includes new headers with type definitions which I
expect will interest genksyms, so don't let it see them.

--- a/net/core/skbuff.c
+++ b/net/core/skbuff.c
@@ -45,8 +45,10 @@
 #include <linux/in.h>
 #include <linux/inet.h>
 #include <linux/slab.h>
+#ifndef __GENKSYMS__
 #include <linux/tcp.h>
 #include <linux/udp.h>
+#endif
 #include <linux/netdevice.h>
 #ifdef CONFIG_NET_CLS_ACT
 #include <net/pkt_sched.h>
